src/components/action-buttons/action-buttons.scss

```diff
@@ -4,7 +4,7 @@
   justify-content: flex-end;
   margin: 1rem 0;
 
-  button {
+  > button {
```

Comment:
This ensures that only `button` elements that are direct children of `.actionButton` are affected by these styles.